version.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# CVE-Update-Job
+# Copyright(C) 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This file carries the version and title of the cve-update-job project."""
+
+__title__ = "cve-update-job"
+__version__ = "0.1.0"
```

Comment:
You can put these directly to `app.py`. No need to have a separate file for this.